README.md

```diff
@@ -159,7 +159,7 @@ The `cache` input is optional, and caching is turned on by default.
 
 The action defaults to search for the dependency file - go.sum in the repository root, and uses its hash as a part of
 the cache key. Use `cache-dependency-path` input for cases when multiple dependency files are used, or they are located
-in different subdirectories.
+in different subdirectories. Wildcards also supported.
```

Comment:
To precise this syntax is not "wildcards" but rather "glob pattern" (https://en.wikipedia.org/wiki/Glob_(programming)). The difference is significant because the latter much more powerful and calling it "wildcards" could be misleading.

Reply:
Fixed.